LocalSettings.php

```diff
@@ -4203,6 +4203,16 @@
 			'bot' => array(
 				'editinterface' => true,
 			),
+			'user' => array(
```

Comment:
@Reception123 by default these permissions are assigned to the '*' group, but I'm revoking them from the '*' group and assigning them to the 'user' group instead (because the '*' group would override the "banned" group)